.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": false
```

Comment:
Can we change this to `"semi": true`?

The whole code should have been prettified already (as I mainly edited this code – prettier is executed automatically in my editor for every edit), just not with explicitly defined settings. It is good to have them now but in order to keep this PR minimal, I prefer if we aligned to the previous implicit setting.

Reply:
Changed. Any suggestion for the `printWidth`? Seems like this also differs from your configuration.